atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/backup/TokenRangeFetcher.java

```diff
@@ -58,22 +56,7 @@ public TokenRangeFetcher(
     }
 
     public Map<InetSocketAddress, RangeSet<LightweightOppToken>> getTokenRange(String tableName) {
-        KeyspaceMetadata keyspaceMetadata = cqlMetadata.getKeyspaceMetadata(namespace);
-        if (keyspaceMetadata == null) {
-            log.error(
-                    "Could not find metadata for a keyspace that is supposed to exist",
-                    SafeArg.of("keyspace", namespace));
-            return ImmutableMap.of();
-        }
-
-        TableMetadata tableMetadata = keyspaceMetadata.getTable(tableName);
-        if (tableMetadata == null) {
-            log.error(
-                    "Could not find metadata for table that is supposed to exist",
-                    SafeArg.of("keyspace", namespace),
-                    SafeArg.of("tableName", tableName));
-            return ImmutableMap.of();
-        }
+        TableMetadata tableMetadata = cqlSession.getTableMetadata(tableName);
```

Comment:
do you need to add retrying logic to any other members of the class?

Reply:
all the table metadata fetching logic should be fed through CqlSession now. We haven't seen the other calls failing.